src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -558,13 +558,13 @@ private static MiscInfoGroup GetOtherInfo(CimSession session)
             // get secure-boot info
             //TODO: Local machine only? Check for that?
             FirmwareType fwType = FirmwareType.Unknown;
-            if (Native.GetFirmwareType(ref fwType))
+            if (GetFirmwareType(ref fwType))
```

Comment:
The `rv.firmwareType` and `rv.physicallyInstalledMemory` properties are nullable and start out as null when `rv` is assigned the newly-constructed `MiscInfoGroup` object. The `GetFirmwareType` and `GetPhysicallyInstalledSystemMemory` methods store their resultant data, if acquired, into _local_ value-type variables and indicate through their return value whether they successfully acquired the data. Only if the return values indicate success are the nullable properties of `rv` assigned the non-null values in the local variables.

That is certainly the intent of this code. Have I missed something?